### PR TITLE
Check output with whitespace

### PR DESCRIPTION
**Pull request by owenmock22**

### CODE REVIEW

**Review by AndrewG2026 — APPROVED** [review]
looks like this fixes the issue with whitespace

File: www/student.html [hunk 1]
```diff
@@ -22,9 +22,14 @@ <h1>Input:</h1>
         <button id="end-button" disabled>End</button>
         <p id="time-displayed"></p>
         <h1>Code Output:</h1>
-        <span id="code-output"></span>
+        <textarea id="code-output"></textarea>
         <h1>Desired Output:</h1>
-        <input type="text" id="target-text" name="target-text" disabled />
+        <textarea
+          type="text"
```
Comment (jncraton) [inline www/student.html:28]: `type = "text"` is not meaningful for a `<textarea>`

**Review by JIckes12 — APPROVED** [review]

File: www/student.html [hunk 2]
```diff
@@ -22,9 +22,14 @@ <h1>Input:</h1>
...
+          id="target-text"
+          name="target-text"
```
Comment (jncraton) [inline www/student.html:30]: As far as I can see, then `name` attribute is never used. Do we need it?

File: www/style.css [hunk 3]
```diff
@@ -35,6 +35,21 @@ sub {
...
+#code-output {
+  width: 500px;
+  height: 175px;
+}
+
+#target-text {
+  width: 500px;
+  height: 175px;
+}
+
```
Comment (jncraton) [inline www/style.css:52]: Should this be a unified style for every `textarea` rather than 3 separate rules?

**Review by jncraton — CHANGES_REQUESTED** [review]

**Pull request merged**